auth/basic/README.md

```diff
@@ -13,4 +13,4 @@ httptransport.NewServer(
 	)
 ```
 
-For AuthMiddleware to be able to pick up Authentication header from a http request we need to pass it through the context with something like ```httptransport.ServerBefore(httptransport.PopulateRequestContext)```
\ No newline at end of file
+For AuthMiddleware to be able to pick up the Authentication header from a HTTP request we need to pass it through the context with something like ```httptransport.ServerBefore(httptransport.PopulateRequestContext)```.
```

Comment:
- [ ] We could go either way, but in American English, it sounds better to my ear to write "an HTTP."